### PR TITLE
Add DotNetSlnRemove alias for dotnet sln remove command

### PR DESCRIPTION
**Pull request by Marusyk**
`dotnet sln remove` - Removes a project or multiple projects from the solution file.

Docs: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-sln#remove

alternative PR for #3817

Please review
Thank you in advance

### CODE REVIEW

File: src/Cake.Common/Tools/DotNet/DotNetAliases.cs [hunk 1]
```diff
@@ -2883,5 +2884,78 @@ public static DotNetPackageList DotNetListPackage(this ICakeContext context, str
             var lister = new DotNetPackageLister(context.FileSystem, context.Environment, context.ProcessRunner, context.Tools);
             return lister.List(project, settings);
         }
+
+        /// <summary>
```
Comment (Marusyk) [inline src/Cake.Common/Tools/DotNet/DotNetAliases.cs:2888]: Does it make sense to consider using partial classes for this DotNetAliases to better grouping and navigation?
e.g.  
DotNetAliases.cs
- DotNetAliases.Sln.cs
- DotNetAliases.Package.cs
- ...
What do you think?

Comment (devlead) [reply src/Cake.Common/Tools/DotNet/DotNetAliases.cs:2888]: Yes, not a bad idea.

Comment (Marusyk) [reply src/Cake.Common/Tools/DotNet/DotNetAliases.cs:2888]: in scope of this PR or create a new one? what do you think?

Comment (devlead) [reply src/Cake.Common/Tools/DotNet/DotNetAliases.cs]: separate PR is probably more clear.

**Review by devlead — APPROVED** [review]
LGTM 👍

**Comment by devlead** [comment]
@Marusyk your changes have been merged, thanks for your contribution 👍

**Pull request merged**